File: server/src/main/java/com/sju18/petmanagement/domain/account/dao/Account.java
package com.sju18.petmanagement.domain.account.dao;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;

import javax.persistence.*;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Account {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "account_id")
    private Long id;

    @Column(unique = true, nullable = false)
    private String username;
    @Column(unique = true, nullable = false)
    private String email;
    @Column(unique = true, nullable = false)
    private String phone;

    @Column(nullable = false)
    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
    private String password;
    @Column(nullable = false)
    private Boolean marketing;

    @Column
    private String nickname;
    @JsonProperty(access = JsonProperty.Access.READ_ONLY)
    private String photoUrl;
    private String userMessage;
}

File: client/android/app/src/main/java/com/sju18001/petmanagement/restapi/dao/Account.kt
package com.sju18001.petmanagement.restapi.dao

data class Account(
    var id: Long,
    val username: String,
    val email: String,
    val phone: String,
    val password: String?,
    val marketing: Boolean?,
    val nickname: String?,
    var photoUrl: String?,
    val userMessage: String?,
    val representativePetId: Long?
)